axiom/nr_php_packages.c

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <math.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "nr_axiom.h"
+#include "nr_limits.h"
+#include "nr_php_packages.h"
+#include "util_memory.h"
+#include "util_minmax_heap.h"
+#include "util_vector.h"
+#include "util_hashmap.h"
+#include "util_hashmap_private.h"
+#include "util_strings.h"
+
+nr_php_package_t* nr_php_package_create(char* name, char* version) {
+  nr_php_package_t* p = NULL;
+
+  if (name == NULL || version == NULL) {
+    return NULL;
+  }
+
+  p = (nr_php_package_t*)nr_malloc(sizeof(nr_php_package_t));
+  if (p == NULL) {
+    return NULL;
+  }
+
+  p->package_name = nr_strdup(name);
+  p->package_version = nr_strdup(version);
+
+  return p;
+}
+
+void nr_php_package_destroy(nr_php_package_t* p) {
+  if (p != NULL) {
+    nr_free(p->package_name);
+    nr_free(p->package_version);
+    nr_free(p);
+  }
+}
+
+void nr_php_packages_add_package(nr_php_packages_t** h, nr_php_package_t* p) {
+  if (h == NULL) {
```

Comment:
All of the conditional statements in your PR need to follow our [style guide](https://github.com/newrelic/newrelic-php-agent/blob/dev/STYLEGUIDE.md) and use 'yoda conditionals'.

Reply:
Fixed: [596aa6b](https://github.com/newrelic/newrelic-php-agent/pull/754/commits/596aa6b81613b00b6d142913cb7c73c8b935d3d3)

Reply:
I think you missed some in the rest of the file(s).

Reply:
I noticed that I missed one in this same file. Fixed here: [64c3366](https://github.com/newrelic/newrelic-php-agent/pull/754/commits/64c336626f1d08e02032ef42bd2ae8be51f22c78). I double checked right now and I don't think the other files have this issue unless I missed it.